esp-alloc/src/lib.rs

```diff
@@ -51,7 +51,28 @@
 //! ```rust
 //! let large_buffer: Vec<u8, _> = Vec::with_capacity_in(1048576, &PSRAM_ALLOCATOR);
 //! ```
-
+//!
+//! You can also gets stats about the heap usage at anytime with:
+//! ```rust
+//! let stats: HeapStats = esp_alloc::HEAP.stats();
+//! // HeapStats implements the Display and defmt::Format traits, so you can pretty print the heap stats.
```

Comment:
Also just a nitpick, feel free to ignore. It was written this way further down in the code, I thought it would be nice to make it the same 😄.
```suggestion
//! // HeapStats implements the Display and defmt::Format traits, so you can pretty-print the heap stats.
```

Reply:
I don't mind, let's make it more consistent :)